src/query/pipeline/sources/src/processors/sources/input_formats/input_context.rs

```diff
@@ -104,7 +173,7 @@ impl InputContext {
         let format = Self::get_input_format(&file_format_options.format)?;
         let file_infos = Self::get_file_infos(&format, &operator, &plan).await?;
         let splits = format.split_files(file_infos, split_size);
-        let rows_per_block = settings.get_max_block_size()? as usize;
+        let rows_per_block = 1000 * 1000 * 0.8 as usize;
```

Comment:
Why hard code `1000 * 1000 * 0.8` here?

Reply:
explained here,  https://github.com/datafuselabs/databend/issues/7760 
use this value to  try to avoid some compact
DEFAULT_ROW_PER_BLOCK = 1000 * 1000, but it is only pub crate， 
since it is temp (should use table`s option later),  I did not change DEFAULT_ROW_PER_BLOCK to pub.

Reply:
I can change it to refer DEFAULT_ROW_PER_BLOCK if you prefer it？

Reply:
I misremembered,  DEFAULT_ROW_PER_BLOCK in crate storage::fuse,  which is not in the deps.


----

in the new commit.
I add a local const https://github.com/datafuselabs/databend/pull/7769/commits/2310d3a96a6ac58393e13d5a274a8d05ae207e2a